robot/Cumulus/resources/NPSP.py

```diff
@@ -1625,8 +1625,6 @@ def click_special_button(self,title):
         In the cases where javascript is being triggered on moving away from field,
         click button doesn't seem to work in headless mode, hence using actionchains moving focus out of field
         and clicking on screen before performing actual click for next element"""
-        actions = ActionChains(self.selenium.driver)
-        actions.move_by_offset(0, 20).click().perform()
```

Comment:
I had to use this for other test cases in GE test suite for a specific reason mentioned in this keyword documentation. Instead of making change to this keyword for your test case, I suggest to use keyword 'Click Gift Entry Button'